### PR TITLE
Blocks: Update fixtures readme and add a check for valid .json in fixture testing

### PR DESCRIPTION
**Pull request by glendaviesnz**
#### Changes proposed in this Pull Request:

* Adds further explanation to the Jetpack blocks fixture tests readme along with an an additional check to automatically check for valid block parsing rather than having to manually check the fixture files.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

- Check out this PR
- In a block with fixture tests, eg. `slideshow` modify one of the `.html` fixture files to have invalid attributes or markup.
- Run `yarn fixtures:regenerate extensions/blocks/slideshow/test/validate.js` and check that error is thrown about an invalid fixture
- Fix the broken markup and run again and make sure no error is thrown

### CODE REVIEW

**Review by andrewserong — DISMISSED** [review]
This is testing nicely for me, and fails as expected when there's invalid markup in a block, or invalid block attributes 👍

Thanks for the additional context in the block-fixtures doc, too, that all reads well and helps clarify how the process works.

File: projects/plugins/jetpack/extensions/shared/test/block-fixtures.md [hunk 1]
```diff
@@ -97,8 +104,7 @@ For all blocks:
 yarn fixtures:regenerate
 ```
 
-After regenerating fixtures, check the diff (using git/github) to check that the changes were expected
-and the block is still valid (`isValid` is `true`).
+After regenerating fixtures, check that the content of the `.parsed.json` and `.parsed.html` files are as expected. An error should be thrown if any of the fixtures are invalid after parsing.
```
Comment (aaronrobertshaw) [inline projects/plugins/jetpack/extensions/shared/test/block-fixtures.md:107]:
```suggestion
After regenerating fixtures, check that the content of the `.parsed.json` and `.serialized.html` files are as expected. An error should be thrown if any of the fixtures are invalid after parsing.
```

Is `.parsed.html` is meant to be `.serialized.html`?

**Review by aaronrobertshaw — COMMENTED** [review]
Tests well for me 👍 

Maybe just a minor tweak is needed in the doc changes.

**Review by andrewserong — APPROVED** [review]
LGTM, thanks for fixing up the docs!

**Review by jeherve — APPROVED** [review]
Thanks for that, that helps!

**Comment by glendaviesnz** [comment]
WP diff committed - rWP222590

**Pull request merged**